### PR TITLE
Fog stack update - combined

### PR DESCRIPTION
**Pull request by ares**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

This combines #3462 and #3515 which can be closed if we go ahead with this one.

### CODE REVIEW

**Comment by tbrisker** [comment]
[test packaging]

**Comment by tbrisker** [comment]
[test packaging]

**Comment by tbrisker** [comment]
[test rpm]

**Comment by tbrisker** [comment]
[test rpm]

**Comment by tbrisker** [comment]
[test rpm]

**Comment by tbrisker** [comment]
[test rpm]

**Comment by tbrisker** [comment]
[test rpm]

**Comment by tbrisker** [comment]
[test rpm]

**Comment by ekohl** [comment]
[test rpm]

**Comment by tbrisker** [comment]
looks like obal linting is failing on rubygem-optimist?

**Comment by dgoetz** [comment]
Yes, it is missing in package_manifest.yaml

**Comment by ares** [comment]
Thanks! Updated the package_manifest.yaml now.

**Comment by tbrisker** [comment]
rbvmomi is still failing to build: http://koji.katello.org/kojifiles/work/tasks/3741/183741/build.log

**Comment by tbrisker** [comment]
Also, not sure why but building rubygem-google-api-client takes an hour: http://koji.katello.org/koji/taskinfo?taskID=183717

**Comment by dgoetz** [comment]
rbvmomi fails because of the files section of the doc package. I updated it and could build it locally.
```
%files doc
%doc %{gem_docdir}
%doc %{gem_instdir}/README.md
%{gem_instdir}/test
```

**Comment by tbrisker** [comment]
@dgoetz thanks for looking into it, could you push the fix to the @ares's branch?

**Comment by dgoetz** [comment]
I also looked into why rubygem-google-api-client took so long.

* Creation of rdoc takes very long `Done installing documentation for google-api-client after 705 seconds`
* Also `/usr/lib/rpm/brp-strip-static-archive /usr/bin/strip` takes quite long
* `Processing files: tfm-rubygem-google-api-client-doc-0.23.9-1.el7.noarch` takes probably so long because of the documentation created

So I looked into the package and it is **145648** files and **35M** only for documentation! Is this worth one hour buildtime or should we disable creation of documentation?

**Comment by dgoetz** [comment]
Build tests have run fine now, but failed at "Notifies GitHub of the status of a Pull Request".

**Comment by ares** [comment]
Thanks @dgoetz! For the google-api-client documentation, I'd say let's skip it. I'm happy to reconsider when I see first report about "I can't read this gem documentation from packages present in Foreman repo.". I have a feeling it will not happen.

**Comment by dgoetz** [comment]
I have disabled docs for google-api-client by resolving `gem_install` and replacing `--document=ri,rdoc` with `--no-document` and commenting the files section. Total build time according to Jenkins was **1m 17s**.

**Comment by evgeni** [comment]
[test rpm]

File: packages/foreman/rubygem-mime-types-data/rubygem-mime-types-data.spec [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Generated from mime-types-data-3.2018.0812.gem by gem2rpm -*- rpm-spec -*-
```
Comment (evgeni) [inline packages/foreman/rubygem-mime-types-data/rubygem-mime-types-data.spec]: we already have mime-types-data in the Rails SCL: https://github.com/theforeman/rails-packaging/tree/tfm-ror52/packages/tfm-ror52-rubygem-mime-types-data

File: packages/foreman/rubygem-mime-types/rubygem-mime-types.spec [hunk 2]
```diff
@@ -0,0 +1,103 @@
+# Generated from mime-types-3.2.2.gem by gem2rpm -*- rpm-spec -*-
```
Comment (evgeni) [inline packages/foreman/rubygem-mime-types/rubygem-mime-types.spec]: we already have mime-types in the Rails SCL: https://github.com/theforeman/rails-packaging/tree/tfm-ror52/packages/tfm-ror52-rubygem-mime-types

**Comment by evgeni** [comment]
```
% koji add-pkg foreman-nightly-rhel7 tfm-rubygem-declarative tfm-rubygem-declarative-option tfm-rubygem-googleauth tfm-rubygem-httpclient tfm-rubygem-memoist tfm-rubygem-optimist tfm-rubygem-os tfm-rubygem-public_suffix tfm-rubygem-representable tfm-rubygem-uber --owner=kojiadm
Adding 10 packages to tag foreman-nightly-rhel7
```

**Pull request merged**